apps/userdashboard/templates/a4_candy_userdashboard/base_userdashboard.html

```diff
@@ -18,33 +18,55 @@
         </div>
     </div>
 
-    <div class="row pt-4 bg-white">
+    <div class="container">
+        <div class="row pt-4">
 
-        <div class="col-md-2 offset-2">
-            <div
-                class="tabpanel userdashboard-profile userdashboard-profile__side"
-                id="userdashboard-user-{{ user.pk }}-tile"
-                role="tabpanel"
-                aria-labelledby="userdashboard-{{ user.pk }}-tile"
-                aria-expanded="false">
-                <a href="{% url 'profile' user.username %}">
-                    {% if user.avatar %}
-                        <img class="userdashboard-profile__avatar mb-2" src="{% thumbnail user.avatar '170x170' crop %}" alt="" />
-                    {% else %}
-                        <img class="userdashboard-profile__avatar mb-2" src="{{ user.avatar_fallback }}" width="170" height="170" alt="" />
-                    {% endif %}
-                </a>
+            <div class="col-md-3">
+                <div
+                    class="tabpanel userdashboard-profile userdashboard-profile__side"
+                    id="userdashboard-user-{{ user.pk }}-tile"
+                    role="tabpanel"
+                    aria-labelledby="userdashboard-{{ user.pk }}-tile"
+                    aria-expanded="false">
+                    <div class="userdashboard-profile__user">
+                        <a href="{% url 'profile' user.username %}">
+                            {% if user.avatar %}
+                                <img class="userdashboard-profile__avatar mb-2" src="{% thumbnail user.avatar '170x170' crop %}" alt="" />
+                            {% else %}
+                                <img class="userdashboard-profile__avatar mb-2" src="{{ user.avatar_fallback }}" width="170" height="170" alt="" />
+                            {% endif %}
+                        </a>
+
+                        <h3 class="mt-0 u-word-break-all">
+                        {{ user.username }}
+                        </h3>
+                        <div>
+                            {% trans 'Since' %} {{ request.user.date_joined|date:"d M Y" }}
+                        </div>
+                    </div>
+                    <div>
+                      <h3>{% trans 'Organisations' %}</h3>
+                      {% for organisation in view.organisations %}
+                      <a class="btn btn--link text-decoration-none text-muted" href="{% url 'a4dashboard:project-list' organisation_slug=organisation.slug %}">
+                        {% if ORGANISATION.logo %}
```

Comment:
the logo is not shown, probably bc it should be 'organisation' instead of ORGANISATION